### DISCUSSION TITLE
Get Access to Open a New Branch

### DISCUSSION THREAD

**Discussion by alexmay48** [Netflix/conductor#3668] [Q&A] [answered]
I have been trying to create a new branch to open a PR for a new feature, but it looks like I don't have permissions to do so. Does someone need to provide that for my account?

**Comment by charybr** [comment, marked as answer]
Create a fork of Netflix/conductor, then create a branch on that fork (since the fork is your repo you will have permission to create branch). Make changes on that branch.  Create a PR (select base repository as Netflix/conductor and base branch main) from that branch to main Netflix/conductor.

**Reply by alexmay48** [reply]
Thanks!
